src/test/kotlin/racingcar/domain/MovingDiscriminatorTest.kt

```diff
@@ -0,0 +1,21 @@
+package racingcar.domain
+
+import org.assertj.core.api.Assertions
+import org.junit.jupiter.api.Test
+
+class MovingDiscriminatorTest {
+
+    @Test
+    fun `숫자 4를 받았을때 전진하는지 확인하는 테스트(4부터 9까지 전진)`() {
```

Comment:
```suggestion
    fun `숫자 4 이상이면 전진한다`() {
```
클래스 자체가 테스트를 위한 클래스이므로, "테스트"라는 문구는 언급하지 않으셔도 될 것 같아요. (이하 관련 내용 동일)
추가로 워딩을 좀 더 다듬어보면 좋을 것 같아요. (이하 관련 내용 동일)